src/main/scala/no/ndla/conceptapi/service/search/SearchConverterService.scala

```diff
@@ -80,6 +81,17 @@ trait SearchConverterService {
       )
     }
 
+    def groupSubjectTagsByLanguage(subjectId: String, tags: List[api.ConceptTags]) =
+      tags
+        .groupBy(_.language)
+        .map {
+          case (lang, conceptTags) => {
+            val tagsForLang = conceptTags.flatMap(_.tags).distinct
```

Comment:
Kan man ha tags uten språk? Hva skjer da isåfall?

Reply:
Nop. Taggene som kobles til et begrep ser sånn ut:
```case class ConceptTags(tags: Seq[String], language: String) extends LanguageField {...}```

Så om de skal være språknøytrale må de ha språk = `unknown`. Da vil de evt komme med om man ber om `unknown` eller `all` språk. Evt om man skrur på `fallback` parameteret.